src/containers/Faq/index.tsx

```diff
@@ -0,0 +1,64 @@
+import React, {FC} from 'react';
+import {HashLink} from 'components';
+
+import './Faq.scss';
+
+interface FaqContent {
+  answer: string;
```

Comment:
change this to `answer: ReactNode` (you gotta import `ReactNode` from react)

Reply:
Done!